Allow changing of distro at build time

rambo opened the pull request:
Ubuntu focal has better arm64 support for example.

rambo committed:
Allow changing of distro at build time

Ubuntu focal has better arm64 support for example.

NyakudyaA commented on Dockerfile:
```diff
@@ -1,7 +1,8 @@
 ARG IMAGE_VERSION=buster
 ARG IMAGE_VARIANT=-slim
-FROM debian:$IMAGE_VERSION$IMAGE_VARIANT
+FROM $DISTRO:$IMAGE_VERSION$IMAGE_VARIANT
```
Can you also add a small section in the README with an example of how to build using the args

rambo replied:
Added.

NyakudyaA replied:
Thanks . Can you fix the failing test by changing the sleep time to 30 in https://github.com/kartoza/docker-postgis/blob/develop/scenario_tests/datadir_init/test.sh

rambo replied:
done, but maybe there's a better way than hardcoded sleep. https://github.com/jwilder/dockerize is a small golang application that can handle waiting for dbs with timeouts etc

NyakudyaA replied:
thanks, We only use the sleep in unit testing. Maybe you could add a PR for this we would be glad to review it

rambo committed:
Document build-arg usage

rambo committed:
increase sleep time

as per https://github.com/kartoza/docker-postgis/pull/233#discussion_r409846863
